config/src/config/network_config.rs

```diff
@@ -29,7 +29,7 @@ use std::{
 // might need to extract into a separate network_constants crate or something.
 pub const HANDSHAKE_VERSION: u8 = 0;
 
-pub type NetworkPeersConfig = HashMap<PeerId, x25519::PublicKey>;
+pub type SeedPubkeySetsConfig = HashMap<PeerId, HashSet<x25519::PublicKey>>;
```

Comment:
I don't think the nomenclature is quite correct in either case...
in practice this will only be used when the remote peers are /not/ on chain discovery, mutual auth, on a unidirectional network, something like a Private(V)FN. I don't imagine we'll have a good case for multiple keys. I would kinda prefer we leave this well enough alone and not modify it, since there really isn't a clear V1 use case that we need to support with it.

Reply:
looking below, I think the term "trusted_peers" and TrustedPeers or TrustedPeersSet is reasonable!